website/docs/dapp/DApp.mdx

```diff
@@ -35,8 +37,19 @@ Welcome to the Nervos CKB developer hub—your go-to for all tools to build dApp
 </Tabs>
 ```
 
-## Smart Contracts Utilization in dApp
-Explore tutorials focused on integrating and utilizing smart contracts within your dApps. Understand the key concepts, protocols, and strategies to enhance your applications, without delving into contract creation or deployment. 
+## Smart Contracts Utilization in DApp
+Explore tutorials focused on integrating and utilizing smart contracts within your dApps. Understand the key concepts, protocols, and strategies to enhance your applications, without delving into contract creation or deployment.
+<CardLayout>
+  <Card title='Create Custom Token' description='Create user-defined tokens using xUDT' href='./DApp/create-custom-token' type='tutorial' />
```

Comment:
I would create a object or file for these text in a separate file.
Good to separate the text and the components so it'll be easier to edit in the future.